temporian/implementation/numpy/operators/sum.py

```diff
@@ -1,8 +1,48 @@
 import numpy as np
 
+from temporian.core.operators.sum import Resolution
 from temporian.implementation.numpy.data.event import NumpyEvent
 
 
 class NumpySumOperator:
+    def __init__(
+        self, resolution: Resolution = Resolution.PER_FEATURE_IDX
+    ) -> None:
+        super().__init__()
+        self.resolution = resolution
+
     def __call__(self, event_1: NumpyEvent, event_2: NumpyEvent) -> NumpyEvent:
-        pass
+        """Sum two Events.
+
+        Args:
+            event_1: First Event.
+            event_2: Second Event.
+            resolution: Resolution of the output Event. PER_FEATURE_IDX sum is done feature index wise. PER_FEATURE_NAME sum is done feature name wise.
+
+        Returns:
+            Sum of the two Events according to resolution.
+
+        Raises:
+            IndexError: If index of both events is not equal.
+            NotImplementedError: If resolution is PER_FEATURE_NAME.
+        """
+        if event_1.data.keys() != event_2.data.keys():
```

Comment:
Think it would be better to check the event's samplings are identical? Since that would also make sure that the timestamps present in each are the same